planning_project:
  ros__parameters:
    num_of_samples: 300
    look_ahead_dist: 4.0
    step_size: 0.3
    goal_sample_rate: 0.2
    angle_sample_range: 1.5708
    safety_padding: 0.3305
    disparity_extend_range: 0.1016
    grid_width: 250
    grid_height: 300
    max_yaw: 1.047197551
    max_vel: 6.0
    grid_resolution: 0.03
    waypoint_file_path: /sim_ws/src/planning_project/waypoints/waypoints.txt
    waypoint_frame_id: map
    pose_to_listen: ego_racecar/base_link